Content.Client/GameTicking/Managers/ClientGameTicker.cs

```diff
@@ -153,8 +162,35 @@ private void RoundEnd(RoundEndMessageEvent message)
             if (_window?.RoundId == message.RoundId)
                 return;
 
-            //This is not ideal at all, but I don't see an immediately better fit anywhere else.
-            _window = new RoundEndSummaryWindow(message.GamemodeTitle, message.RoundEndText, message.RoundDuration, message.RoundId, message.AllPlayersEndInfo, _entityManager);
+            //Cyberfinn changes:
+            //back up the round info:
+            _roundEndContainer._message = message;
+            //display the window
+            DisplayRoundEndSummary(_roundEndContainer._message);
```

Comment:
first comment say back it up, to show it later when requested
second comment is obvious so remove

Reply:
I like to comment code for those that dont know coding to check the logic, but will remove as suggested. Thanks man!

Reply:
I understand commenting code like this, but in this case the code is bassicly self explaining, I would assume "DisplayRoundEndSummary" displays the round end summery just from the name

Reply:
lol true. Will remove as suggested, thanks again! 
Should I add a new PR for the updated code, or what?